pkg/agent/cache/manager.go

```diff
@@ -103,23 +104,23 @@ type Manager interface {
 
 type cacheManager struct {
 	jobs.Job
-	logger                  log.FieldLogger
-	apiMap                  cache.Cache
-	instanceMap             cache.Cache
-	categoryMap             cache.Cache
-	managedApplicationMap   cache.Cache
 	accessRequestMap        cache.Cache
-	subscriptionMap         cache.Cache
-	sequenceCache           cache.Cache
-	resourceCacheReadLock   sync.Mutex
-	cacheLock               sync.Mutex
-	persistedCache          cache.Cache
-	teams                   cache.Cache
+	apiMap                  cache.Cache
 	ardMap                  cache.Cache
-	crdMap                  cache.Cache
 	cacheFilename           string
+	cacheLock               sync.Mutex
+	categoryMap             cache.Cache
+	crdMap                  cache.Cache
 	hasLoadedPersistedCache bool
+	instanceMap             cache.Cache
 	isCacheUpdated          bool
+	logger                  log.FieldLogger
+	managedApplicationMap   cache.Cache
+	persistedCache          cache.Cache
+	resourceCacheReadLock   sync.Mutex
+	sequenceCache           cache.Cache
+	subscriptionMap         cache.Cache
+	teams                   cache.Cache
```

Comment:
Not sure if this is something goland did for you or not, but I once read an article that talked about how golang reserves memory for attributes in a struct.  

It does not matter for this, but something interesting to keep in mind in the future. 

https://medium.com/techverito/golang-struct-size-and-memory-optimisation-b46b124f008d